packages/common/src/ui/layout/tables/context/TableContext.ts

```diff
@@ -152,6 +164,60 @@ export const createTableStore = (): UseStore<TableStore> =>
         };
       });
     },
+    setFocusDown: () => {
```

Comment:
The difference between these two is just the adding to the current idx, I think?

Did it become really hard to read we consolidated?

Reply:
Yeah, better to combine together with a parameter for up/down? I agree.

Done 4a27af25